spec/xclarity_client_storage_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe XClarityClient do
+  before :all do
+    WebMock.allow_net_connect! #-- Uncomment this line if you're testing with a external mock.
+  end
+
+  let(:conf) do
+    XClarityClient::Configuration.new(
+      username:   ENV['LXCA_USERNAME'],
+      password:   ENV['LXCA_PASSWORD'],
+      host:       ENV['LXCA_HOST'],
+      port:       ENV['LXCA_PORT'],
+      auth_type:  ENV['LXCA_AUTH_TYPE'],
+      verify_ssl: ENV['LXCA_VERIFY_SSL']
+    )
+  end
+
+  let(:client) { XClarityClient::Client.new conf }
+
+  it 'has a version number' do
+    expect(XClarityClient::VERSION).not_to be nil
+  end
+
+  context 'with discover_storages' do
+    before do
+      @storages = client.discover_storages
+    end
+
+    describe 'GET /storage' do
+      it 'should respond with an array' do
+        expect(@storages.class).to eq(Array)
+        expect(@storages.any?).to eq(true)
+      end
+
+      it 'correct response values' do
+        expect(@storages[0].name).to eq('S3200-1')
+        expect(@storages[0].type).to eq('Lenovo Storage')
+        expect(@storages[0].accessState).to eq('Online')
+        expect(@storages[0].driveBays).to eq(12)
+        expect(@storages[0].enclosureCount).to eq(1)
+        expect(@storages[0].canisterSlots).to eq(2)
+      end
+    end
+  end
+
+  context 'with fetch_storages' do
+    before do
+      @include_attributes = %w[name type accessState driveBays enclosureCount canisterSlots room]
+      @exclude_attributes = %w[type room]
+      @uuids = client.discover_storages.map { |storage| storage.uuid }
+    end
+
+    it 'missing attributes should not be nil' do
+      storages = client.fetch_storages(@uuids, @include_attributes)
+      expect(storages[0].room).not_to be_nil
+    end
+
+    it 'excluded attributes should not be nil' do
+      storages = client.fetch_storages(@uuids, nil, @exclude_attributes)
+      expect(storages[0].room).not_to be_nil
```

Comment:
__Required:__
Once the `room` attribute is in `@exclude_attributes` it should be nil.
Also, as the suggestion before, could you add a test for the attributes that is not in `@exclude_attributes`, testing the exactly value.